Common/src/main/java/mezz/jei/common/color/ColorName.java

```diff
@@ -0,0 +1,4 @@
+package mezz.jei.common.color;
+
+public record ColorName(String name, Integer color) {
```

Comment:
Is there a reason for having the color be an `Integer` as opposed to an `int`?

Reply:
hm no good reason, I should change it